dev/Scroller/Scroller.h

```diff
@@ -276,8 +288,14 @@ class Scroller :
         ScrollerDimension dimension);
     void EnsurePositionBoundariesExpressionAnimations();
     void EnsureTransformExpressionAnimations();
-    void SetupSnapPoints(std::set<winrt::ScrollerSnapPointBase, winrtProjectionComparator>* snapPoints, ScrollerDimension dimension);
-    void FixSnapPointRanges(std::set<winrt::ScrollerSnapPointBase, winrtProjectionComparator>* snapPoints);
+    void SetupSnapPoints(
+        _In_opt_ std::set<winrt::ScrollSnapPointBase, winrtProjectionComparator>* scrollSnapPointsSet,
```

Comment:
>_In_opt_ [](start = 8, length = 8)

I thought we weren't using these _In_opt_  things anymore (I forget their proper name).

Reply:
Could get rid of them.